cmd/agent-mobile/pkg/wrappers/command/didclient.go

```diff
@@ -68,6 +68,22 @@ func (de *DIDClient) ResolveWebDIDFromOrbDID(request *models.RequestEnvelope) *m
 	return &models.ResponseEnvelope{Payload: response}
 }
 
+// VerifyWebDIDFromOrbDID verify web DID from orb DID.
```

Comment:
no need of unit tests?

Reply:
https://github.com/trustbloc/agent-sdk/pull/423/files#diff-58278360b56af45f7f792b29051f3105f9719d926ded56130dea1ba10ea272d0R210